### PR TITLE
Fixes #14496 - make the job template name display limit higher

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/views/job_templates/index.html.erb [hunk 1]
```diff
@@ -19,7 +19,7 @@
   <tbody>
     <% for job_template in @templates %>
       <tr>
-        <td class="display-two-pane"><%= link_to_if_authorized trunc_with_tooltip(job_template),
+        <td class="display-two-pane"><%= link_to_if_authorized trunc_with_tooltip(job_template, 120),
```
Comment (tbrisker) [inline app/views/job_templates/index.html.erb]: why not add 'ellipsis' to the td class and remove the `trunc_with_tooltip`?

**Comment by ares** [comment]
good idea, changed to ellipsis

**Comment by dLobatog** [comment]
:+1:

**Comment by stbenjam** [comment]
Way better.  ACK.

**Comment by stbenjam** [comment]
Merging now, so I can try to convince downstream it's worthwhile to pull in.

**Pull request merged**